src/games/supported/Boxing.cpp

```diff
@@ -123,3 +125,11 @@ void BoxingSettings::loadState(Deserializer & ser) {
   m_terminal = ser.getBool();
 }
 
+DifficultyVect BoxingSettings::getAvailableDifficulties(){
+    DifficultyVect diff;
+    diff.push_back(0);
```

Comment:
Isn't the second bit for the second player? There's only two difficulty choices per player (A and B).